data-model/src/test/java/org/pdxfinder/graph/dao/TreatmentProtocolTest.java

```diff
@@ -0,0 +1,295 @@
+        TreatmentComponent drug1 = new TreatmentComponent();
+        TreatmentComponent drug2 = new TreatmentComponent();
+
+        control.setDose(CONTROL_DOSE_1);
+        drug1.setDose(DRUG_DOSE_2);
+        drug2.setDose(DRUG_DOSE_3);
+
+        control.setDuration(CONTROL_DURATION_1);
+        drug1.setDuration(DRUG_DURATION_2);
+        drug2.setDuration(DRUG_DURATION_3);
+
+        control.setType("Control");
+        drug1.setType("Drug");
+        drug2.setType("Drug");
+
+        control.setTreatment(treatments.get(0));
+        drug1.setTreatment(treatments.get(1));
+        drug2.setTreatment(treatments.get(2));
+
+        componentsList.add(control);
+        componentsList.add(drug1);
+        componentsList.add(drug2);
+
+        Response testResponse = new Response(RESPONSE_STRING, treatmentProtocol);
+
+        treatmentProtocol.setComponents(componentsList);
+        treatmentProtocol.setResponse(testResponse);
+
+        return treatmentProtocol;
+    }
+
+    private void createMockDatasets2() {
+
+        Treatment treatment;
+        treatment = createTreatmentsAndOntologyRelationship();
+        treatmentProtocolList.addAll(createTreatmentComponentsAndProtocols2(treatment));
+    }
+
+    private Treatment createTreatmentsAndOntologyRelationship() {
+
+        Treatment treatment1 = new Treatment(FIRST_TREATMENT_NAME);
+        TreatmentToOntologyRelationship ontologyRelationship1 = new TreatmentToOntologyRelationship();
+        treatment1.setTreatmentToOntologyRelationship(ontologyRelationship1);
+
+        return treatment1;
+    }
+
+    private List<TreatmentProtocol> createTreatmentComponentsAndProtocols2(Treatment treatments){
+
+        TreatmentProtocol treatmentProtocol = new TreatmentProtocol();
+        TreatmentProtocol treatmentProtocol2 = new TreatmentProtocol();
+
+        List<TreatmentComponent> componentsList1 = new ArrayList<>();
+        List<TreatmentComponent> componentsList2 = new ArrayList<>();
+
+        TreatmentComponent control = new TreatmentComponent();
+        TreatmentComponent drug1 = new TreatmentComponent();
+
+        control.setType("Control");
+        control.setTreatment(treatments);
+
+        control.setDose(CONTROL_DOSE_1);
+        control.setDuration(CONTROL_DURATION_1);
+
+        componentsList1.add(control);
+        componentsList2.add(drug1);
+
+        Response testResponse = new Response(RESPONSE_STRING, treatmentProtocol);
+        Response testResponse2 = new Response(RESPONSE_STRING, treatmentProtocol2);
+
+        treatmentProtocol.setComponents(componentsList1);
+        treatmentProtocol2.setComponents(componentsList2);
+
+        treatmentProtocol.setResponse(testResponse);
+        treatmentProtocol2.setResponse(testResponse2);
+
+        return Arrays.asList(treatmentProtocol,treatmentProtocol2);
+    }
+
+    @Test
+    public void Given_getTreatmentString_When_MultipleTCandControlsOn_Then_ReturnNotNullConcatenatedString(){
```

Comment:
Needs to be renamed so naming convention is consistent - Compare `_getTreatment` and `_MultipleTC`. Same issue in other test names